recipes/service.rb

```diff
@@ -9,30 +9,35 @@
   node['rundeck_bridge']['options'].map { |key, value| "--#{key} #{value}" },
 ].join(' ')
 
-
-# Overwrite and update the command
-# (Workaround for Upstart and logs)
-node['rundeck_bridge']['poise_service']['options'].each do |k, v|
-  case k
-  when 'upstart'
-    node.default['rundeck_bridge']['poise_service']['options'][k]['command'] = "#{command} 2>&1 > #{node['rundeck_bridge']['home']}/server.log"
-  when 'systemd'
-    node.default['rundeck_bridge']['poise_service']['options'][k]['command'] = command
-  else
-    fail "Unsupported init service #{k}"
-  end
+systemd_unit 'chef-rundeck.service' do
+  action %i[create enable]
+  content(Unit:    {
+    Description: 'chef-rundeck',
+  },
+  Service: {
+    Environment: '',
+    ExecStart: command,
+    ExecReload: '/bin/kill -HUP $MAINPID',
+    KillSignal: 'TERM',
+    User: 'chef-rundeck',
+    WorkingDirectory: '/var/log/chef-rundeck',
+    Restart: 'on-failure',
+  },)
 end
 
+=begin
```

Comment:
Shouldn't this comment block be cleaned up ?

Reply:
Indeed 👍